staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3782,10 +3782,13 @@ type NodeSpec struct {
 	// ID of the node assigned by the cloud provider in the format: <ProviderName>://<ProviderSpecificNodeID>
 	// +optional
 	ProviderID string `json:"providerID,omitempty" protobuf:"bytes,3,opt,name=providerID"`
-	// Unschedulable controls node schedulability of new pods. By default, node is schedulable.
-	// More info: https://kubernetes.io/docs/concepts/nodes/node/#manual-node-administration
+
+	// Deprecated. TaintNodeByCondition has replaced the existing node condition. After that
```

Comment:
We can deprecate this only when TaintNodeByCondition is GA. Then we must wait for 1 year after deprecation before we can remove this from our API.

Reply:
ok, got it, thanks! I will keep an eye on this though :)